server/src/packages/pas-form/pas-form.service.ts

```diff
@@ -43,16 +43,24 @@ export class PasFormService {
           dcp_pasform
       `);
 
-      // if pasA is of a higher version than pasB, it should come first
-      const [{ dcp_pasform: latestPasForm }] = pasPackages.sort((pasA, pasB) => {
-        return pasB.dcp_packageversion - pasA.dcp_packageversion;
-      });
+      if (!pasPackages) {
+        // if pasA is of a higher version than pasB, it should come first
+        const [{ dcp_pasform: latestPasForm }] = pasPackages.sort((pasA, pasB) => {
+          return pasB.dcp_packageversion - pasA.dcp_packageversion;
+        });
 
-      return latestPasForm;
+        return latestPasForm;
+      }
+
+      return null;
```

Comment:
@NYCPlanning/engineering I thought about returning a blank object `{}` here instead of a null. Or maybe an object with null values for the expected fields. e.g.
```
return {
  dcp_projectdescriptionproposedarea: null, 
 dcp_projectdescriptionproposeddevelopment: null,
}
```
But ultimately think `null` most clearly conveys to consumers of this function that no results were found. Thoughts?

Reply:
I think your instincts are right. It make sense that asking for PAS values would return `null` because a PAS doesn't exist.